modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/BaseObjectEntryRelatedObjectsResourceImpl.java

```diff
@@ -47,6 +48,31 @@
  */
 public abstract class BaseObjectEntryRelatedObjectsResourceImpl {
 
+	@DELETE
+	@Parameters(
+		{
+			@Parameter(in = ParameterIn.PATH, name = "currentObjectEntryId"),
+			@Parameter(in = ParameterIn.PATH, name = "objectRelationshipName"),
+			@Parameter(in = ParameterIn.PATH, name = "relatedObjectEntryId")
+		}
+	)
+	@Path(
+		"/{currentObjectEntryId}/{objectRelationshipName}/{relatedObjectEntryId}"
+	)
+	@Produces({"application/json", "application/xml"})
+	@Tags(@Tag(name = "ObjectEntry"))
+	public abstract void deleteCurrentObjectEntry(
```

Comment:
This name is really weird for the action that we want to execute. I know it comes from the REST Builder as it generates the operationId, but in this case, I think we should do something with this in the short future (to avoid possible breaking change in the future)